### ISSUE TITLE
Bazarr only accessible via localhost/loopback address

### ISSUE THREAD

**Issue by 64Hz** [morpheus65535/bazarr#639]
**Describe the bug**
Bazarr web interface only accessible on Localhost/loopback address. (127.0.0.1/localhost).

**To Reproduce**
1. Forward bazarr port in Router (6767 by default).
2. Allow ports in Windows Firewall for 6767 incoming.
3. Set listening address in Bazarr to 0.0.0.0 and restart.
4. Try and access Bazarr through external IP/DNS name. 

The above works for Sonarr and Radarr, but Bazarr will timeout when I try to connect.

**Expected behavior**
Access Bazarr through external IP / DNS name, similar to Sonarr and Radarr.

**Screenshots**
N/A

**Software (please complete the following information):**
Bazarr Version: 0.8.2.4
Sonarr Version: 2.0.0.5338
Radarr Version: 0.2.0.1358
OS: Windows 10

**Additional context**
I duplicated the setup I used for my Sonarr/Radarr set up for Bazarr and no luck. Will provide any details required to resolve this.

**Comment by morpheus65535** [comment]
Can you access it from your Win10 computer using local address (192.x or 10.x)? And from another computer on your local network using the same IP?

**Comment by 64Hz** [comment]
I cannot, no.. I thought that was weird

**Comment by 64Hz** [comment]
Sorry forgive me, I can access it from my local computer on 192.168.x.x:6767, but not another device on the same network.

**Comment by morpheus65535** [comment]
Then you have an issue with your local firewall blocking traffic or your router not properly forwarding traffic to your computer.

**Comment by 64Hz** [comment]
Yeah, that's on me. I disabled firewall and it worked.. I'll resolve it, sorry!
